src/app/(protected)/notebooks/page.tsx

```diff
@@ -60,9 +97,12 @@ export default function Page() {
           <div>
             <h1 className='text-2xl pl-2 pb-8 font-bold'>My Notebooks</h1>
             <div className='flex flex-row grow'>
-            <SearchBar onChange={(e) => doFilter(e.currentTarget.value)} />              
-            <div className='flex flex-row grow justify-end'>
-                <button type="button" className="button bg-new font-medium rounded-lg text-sm px-10 py-2">New</button>
+              <SearchBar onChange={(e) => doFilter(e.currentTarget.value)} />
+              <div className='flex flex-row grow justify-end'>
+                <CreateNotebookCard
+                  buttonTitle="New"
+                  onSubmit={(title, description) => createNotebook({ variables: { title, description, user } })}
```

Comment:
What is there a `user` passed here but not in the next one? Should the user be passed? I feel like it's always the current logged in user that will be passed and thus could simply be handled by the function. 

https://github.com/Code-Society-Lab/cursif-web/pull/37/files#diff-0a17f7dcbc229f8cdbe0ba392d30622af65cc66b13f7fbec8d712299ae06262eR120